gen/util.js

```diff
@@ -18,7 +18,12 @@ function hex(n, pad)
 
 function mkdirpSync(dir)
 {
-    fs.mkdirSync(dir, { recursive: true });
+    let stat = fs.statSync(dir, { throwIfNoEntry: false });
+    if (stat === undefined) {
+        fs.mkdirSync(dir, { recursive: true });
+    } else if (!stat.isDirectory()) {
+        throw 'path exists but is not dir';
+    }
```

Comment:
Should be written in [EAFP](https://docs.python.org/3/glossary.html#term-eafp) style. Please also use the same style as the surrounding code (double quotes, newline before `{`, `const` over `let`, etc.). Personally, I think the existing error is good enough, so feel free to remove this commit.